client/README.md

```diff
@@ -35,7 +35,47 @@ You can then use the following command to start the server on your machine:
 npm start
 ```
 
-### Usage
+### StoryBook Organization
+
+Branch: UpdateStorybookReadme
+
+- Layout
+
+Anything pertaining to components that might be set on a page. Example: Icons, Logos, Profile picture's or Footers.
+
+- Banners
+
+Speak for themselves. Used on the Home page for "Welcome back" or Community Banner with an Input on it
+
+- Cards
+
+Anything with the name "Card" in it will be stored here. Pertains to things such as Media Cards, Upcoming event cards, ETC.
+
+- Atoms/Atomic Components
+
+An atomic component would be a button or a text input
+
+- (Not Implemented Yet) Molecule Components
+
+A molecule would be a SearchInput component that uses both the button and text input Atoms. If you create a Molecule please list it as such.
+
+- Pages
+
+These are the physical full pages that have been put together.
+
+## PROPER SYNTAX FOR STORYBOOK
```

Comment:
Can you lowercase the heading, just capitalize the important words( `Proper Syntax for Storybook`)